IoTCoreDefaultApp/IoTOnboardingService/OnboardingService.cs

```diff
@@ -237,7 +237,7 @@ public IAsyncOperation<OnboardingConfigureWifiResult> ConfigureWifiAsync(AllJoyn
             return Task.Run(() =>
             {
                 // make sure a valid value is provided for auth type
-                if (!Enum.IsDefined(typeof(AuthType), interfaceMemberAuthType))
+                if (!Enum.IsDefined(typeof(AuthType), (AuthType)interfaceMemberAuthType))
```

Comment:
why are we casting interfaceMemberAuthType?  isn't the point of Enum.IsDefined to protect this sort of cast?